feed/consts.py

```diff
@@ -0,0 +1,12 @@
+from enum import Enum
+
+
+class FeedVersion(str, Enum):
+    RSS_2_0 = "rss-2.0"
```

Comment:
It would be nice to also support RSS versions 0.91, 1.0 per classic. I hadn't created an issue for that yet so you had no way of knowing, but it's also not a very high priority.

Reply:
I see that you've created an issue. I agree with this. But then how would you envision the url scheme? I would keep the `/rss/` and `/atom/` urls and have them default to 2.0 and 1.0 respectively (actually the latest version) and then allow a header `VERSION` that can be one of the (0.91, 1.0, 2.0) for RSS and 1.0 for Atom.

I would like to avoid version in the query parameters and url.

Reply:
I think what you are proposing sounds fine. Doing a quick scan of our logs, most clients are explicitly requesting `version=2.0` when that parameter is provided.

Reply:
Added support for version header.